src/main/java/edu/hm/hafner/analysis/parser/PyLintParser.java

```diff
@@ -56,21 +81,23 @@ private Severity mapPriority(final String category) {
             // if the category is missing from the output, default to 'normal'.
             return Severity.WARNING_NORMAL;
         }
-        switch (category.charAt(0)) {
+        switch (category) {
+            // [I]nformational messages that Pylint emits (do not contribute to your analysis score)
             // [R]efactor for a ?good practice? metric violation
             // [C]onvention for coding standard violation
-            case 'R':
-            case 'C':
+            case "I":
+            case "R":
+            case "C":
                 return Severity.WARNING_LOW;
 
             // [W]arning for stylistic problems, or minor programming issues
-            case 'W':
+            case "W":
                 return Severity.WARNING_NORMAL;
 
             // [E]rror for important programming issues (i.e. most probably bug)
             // [F]atal for errors which prevented further processing
-            case 'E':
-            case 'F':
+            case "E":
+            case "F":
                 return Severity.WARNING_HIGH;
```

Comment:
We also have `Severity.ERROR`. Would it make sense to use it as well?

Reply:
That is a really good idea.

Reply:
Done